wp-content/themes/bh/scss/_responsive.scss

```diff
@@ -2554,10 +2587,28 @@
 					height: 175px;
 				}
 			}
-			
+
+			h2.event-name {

```

Comment:
Structure should be follow exactly the same structure of event.scss, meaning h2.event-name should be declared, in this case, before .col-3